Specs/DateTime/German/DatePeriodParser.json

```diff
@@ -23,26 +23,26 @@
       }
     ]
   },
-  {//Assert.Null
-    "Input": "Ich bin nächsten Monat vom 4. bis zum 23. weg",
```

Comment:
Why was "nächsten Monat" removed? Will support for relative day/month/etc come in a follow up PR?

Reply:
The Phrase "Nächsten Monat" is fully working, just went further down (Or is now in DateTimeModel). I removed it to quickly test the DateRange without a specific month given, which did not work at first.
I can add it as separate testcase.

Reply:
Please add as a separate case and other similar situations.
Thanks.